x-pack/plugins/synthetics/common/rules/legacy_uptime/translations.ts

```diff
@@ -0,0 +1,160 @@
+    defaultMessage: 'Uptime TLS',
+  }),
+  description: i18n.translate('xpack.synthetics.alerts.tls.description', {
+    defaultMessage: 'Alert when the TLS certificate of an Uptime monitor is about to expire.',
+  }),
+};
+
+export const TlsTranslationsLegacy = {
+  defaultActionMessage: i18n.translate('xpack.synthetics.alerts.tls.legacy.defaultActionMessage', {
+    defaultMessage: `Detected {count} TLS certificates expiring or becoming too old.
+{expiringConditionalOpen}
+Expiring cert count: {expiringCount}
+Expiring Certificates: {expiringCommonNameAndDate}
+{expiringConditionalClose}
+{agingConditionalOpen}
+Aging cert count: {agingCount}
+Aging Certificates: {agingCommonNameAndDate}
+{agingConditionalClose}
+`,
+    values: {
+      count: '{{state.count}}',
+      expiringCount: '{{state.expiringCount}}',
+      expiringCommonNameAndDate: '{{state.expiringCommonNameAndDate}}',
+      expiringConditionalOpen: '{{#state.hasExpired}}',
+      expiringConditionalClose: '{{/state.hasExpired}}',
+      agingCount: '{{state.agingCount}}',
+      agingCommonNameAndDate: '{{state.agingCommonNameAndDate}}',
+      agingConditionalOpen: '{{#state.hasAging}}',
+      agingConditionalClose: '{{/state.hasAging}}',
+    },
+  }),
+  name: i18n.translate('xpack.synthetics.alerts.tls.legacy.clientName', {
+    defaultMessage: 'Uptime TLS (Legacy)',
+  }),
+  description: i18n.translate('xpack.synthetics.alerts.tls.legacy.description', {
+    defaultMessage:
+      'Alert when the TLS certificate of an Uptime monitor is about to expire. This alert will be deprecated in a future version.',
+  }),
+};
+
+export const DurationAnomalyTranslations = {
+  defaultActionMessage: i18n.translate(
+    'xpack.synthetics.alerts.durationAnomaly.defaultActionMessage',
+    {
+      defaultMessage: `Abnormal ({severity} level) response time detected on {monitor} with url {monitorUrl} at {anomalyStartTimestamp}. Anomaly severity score is {severityScore}.
+Response times as high as {slowestAnomalyResponse} have been detected from location {observerLocation}. Expected response time is {expectedResponseTime}.`,
+      values: {
+        severity: '{{context.severity}}',
+        anomalyStartTimestamp: '{{context.anomalyStartTimestamp}}',
+        monitor: '{{context.monitor}}',
+        monitorUrl: '{{{context.monitorUrl}}}',
+        slowestAnomalyResponse: '{{context.slowestAnomalyResponse}}',
+        expectedResponseTime: '{{context.expectedResponseTime}}',
+        severityScore: '{{context.severityScore}}',
+        observerLocation: '{{context.observerLocation}}',
+      },
+    }
+  ),
+  defaultRecoveryMessage: i18n.translate(
+    'xpack.synthetics.alerts.durationAnomaly.defaultRecoveryMessage',
+    {
+      defaultMessage: `Alert for abnormal ({severity} level) response time detected on monitor {monitor} with url {monitorUrl} from location {observerLocation} at {anomalyStartTimestamp} has recovered`,
```

Comment:
```suggestion
      defaultMessage: `The alert for abnormal ({severity} level) response time detected at {anomalyStartTimestamp} on the monitor {monitor} checking {monitorUrl} from {observerLocation} is no longer active.`,
```
Wondering if we couldn't say something simpler like "The abnormal response time detected at {anomalyStartTimestamp} on the monitor {monitor} is now back to normal."